psql2mysql.py

```diff
@@ -43,6 +43,15 @@ def getTables(self):
         metadata.reflect()
         return metadata.tables
 
+
+    # adapt given query so it excludes deleted items
+    def _exclude_deleted(self, table, query):
+        if not "deleted" in table.columns:
+            return query
+        if isinstance(table.columns["deleted"].type, types.INTEGER):
```

Comment:
Is does this really work? It seems that e.g. manila is using "varchar" for some if the "deleted" columns for whatever reason.

Reply:
Oh, manila tables really have them all, `deleted` as Boolean, Integer and String :-)